### ISSUE TITLE
File names wrong when doing clean installation

### ISSUE THREAD

**Issue by lhilton** [nylo-core/laravel_auth_slate#2]
Hello,

I performed a clean installation of Nylo, then installed this slate. My Nylo installation was with [v5.28.0](https://github.com/nylo-core/nylo/releases/tag/v5.28.0).

```bash
git clone https://github.com/nylo-core/nylo.git testapp
cd testapp
flutter pub get
open -a Simulator --args -CurrentDeviceUDID $(xcrun simctl list devices | grep 'iPhone 15' | awk
 '{print $NF}' | head -n1)
flutter run
```

This resulted in the standard boiler plate Nylo app demo app showing up in the simulator and working as expected.

Next I performed a clean installation of `laravel_auth_slate` (v1.0.25 at this time) via:

```bash
dart pub add laravel_auth_slate
metro publish:slate laravel_auth_slate
```

I then performed the two manual corrections noted inside of `config/events.dart` and `routes/router.dart`. I also re-ran `flutter pub get` to make sure I had any dependencies down.

At this point, I reran `flutter run`. My simulator was still open. I get the following error output:

```
> flutter run
Launching lib/main.dart on iPhone 15 Pro Max in debug mode...
Running Xcode build...                                                  
Xcode build done.                                            2.8s
Failed to build iOS app
Error (Xcode): lib/config/decoders.dart:3:8: Error: Error when reading
'lib/app/controllers/forgot_password_controller.dart': No such file or
directory


Could not build the application for the simulator.
Error launching application on iPhone 15 Pro Max.
```

This message is correct, the file named `lib/app/controllers/forgot_password_controller.dart` is missing. Instead, the file `lib/app/controllers/forgot_password_controller_controller.dart` exists.

Notice the extra `_controller` in the filename.

Running `git status` shows me that several files have suspicious names with duplicated suffixes. Specifically in the `controller` and `page` sub-folders.

Manually renaming the files with duplicated suffixes seems to be the correct course of action. It at least changed the error message to one I will ticket out as a separate issue.

**Comment by agordn52** [comment]
Thanks for reporting this @lhilton, I'll get this resolved today.

**Comment by agordn52** [comment]
Hi @lhilton,

This is now resolved now in laravel_auth_slate [1.1.1](https://pub.dev/packages/laravel_auth_slate).

Now it won't add duplicated **suffixes**, can you try this in a fresh Nylo project to confirm?
I will close the ticket once it's definitely resolved 👍

**Comment by agordn52** [comment]
Closing this issue, resolved in [1.1.1](https://pub.dev/packages/laravel_auth_slate).
